lib/v2/layoffs/index.js

```diff
@@ -4,22 +4,8 @@ const { parseDate } = require('@/utils/parse-date');
 const ROW_COUNT = 100;
 
 const WEBSITE_URL = 'https://layoffs.fyi';
-const SOURCE_URL = `https://airtable.com/v0.3/view/viwA14Z1pM69YIsaW/readSharedViewData?
-stringifiedObjectParams=%7B%22shouldUseNestedResponseFormat%22%3Atrue%7D&
-requestId=reqQcci3rVyhkcAyX&accessPolicy=%7B%22allowedActions%22%3A%5B%7B%22
-modelClassName%22%3A%22view%22%2C%22modelIdSelector%22%3A%22viwA14Z1pM69YIsaW%22%2C%22
-action%22%3A%22readSharedViewData%22%7D%2C%7B%22modelClassName%22%3A%22view%22%2C%22
-modelIdSelector%22%3A%22viwA14Z1pM69YIsaW%22%2C%22action%22%3A%22getMetadataForPrinting
-%22%7D%2C%7B%22modelClassName%22%3A%22view%22%2C%22modelIdSelector
-%22%3A%22viwA14Z1pM69YIsaW%22%2C%22action%22%3A%22readSignedAttachmentUrls
-%22%7D%2C%7B%22modelClassName%22%3A%22row%22%2C%22modelIdSelector%22%3A%22
-rows%20*%5BdisplayedInView%3DviwA14Z1pM69YIsaW%5D%22%2C%22action%22%3A%22
-createBoxDocumentSession%22%7D%2C%7B%22modelClassName%22%3A%22row%22%2C%22
-modelIdSelector%22%3A%22rows%20*%5BdisplayedInView%3DviwA14Z1pM69YIsaW%5D%22%2C%22
-action%22%3A%22createDocumentPreviewSession%22%7D%5D%2C%22shareId%22%3A%22shrqYt5kSqMzHV9R5%22%2C%22
-applicationId%22%3A%22app1PaujS9zxVGUZ4%22%2C%22generationNumber%22%3A0%2C%22
-expires%22%3A%222023-02-02T00%3A00%3A00.000Z%22%2C%22
-signature%22%3A%22096478e17705ffdac261da07c5b4c025aff6d8c6d86be027533fd248913798de%22%7D`;
+const SOURCE_URL =
```

Comment:
Please parse this URL programmatically. I don't think you like updating it every month.

You can obtain it from `window.__stashedPrefetch.urlWithParams` in `https://airtable.com/embed/shrqYt5kSqMzHV9R5`

Reply:
Sounds good, I will post a PR tomorrow, thanks for the advice!!

Reply:
Added a cache to cache the data source link and re-fetch it when the URL is invalid.